docs/user/tutorials/build.md

```diff
@@ -0,0 +1,170 @@
+# Build
+
+This software is a plugin extension to Odin data and requires that you install and build both Odin-data and Odin-control to be able to utilize this project. This page will provide instructions for installing these as well as the Aravis library.
+
+:::{note}
+This tutorial has been written for Ubuntu 22.4 Jellyfish. Compatibility with other Linux systems isn't guaranteed.
+:::
+
+## External Software Dependencies
+
+The plugin has the following direct dependencies:
+
+- [Aravis v0.8.31](https://github.com/AravisProject/aravis): used to interface with genicam cameras.
+- [Cmake v2.9](https://cmake.org/): used to generate the make files.
```

Comment:
Is this particular version of cmake defined by Odin somehow? It's a really *really* old version. What is now known as "modern" cmake is v3+

Reply:
Changed the link and version (it's 2.8) to better reflect the actual version used.